javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/declarations/AstResolutionUtils.java

```diff
@@ -41,10 +41,6 @@
  */
 class AstResolutionUtils {
 
-    static AccessSpecifier toAccessLevel(EnumSet<Modifier> modifiers) {
```

Comment:
We may still need this as the AccessSpecifier is different that the modifiers. Some modifiers have nothing to do with access and the default access is not represented by modifiers